### PR TITLE
Add recipe for Mad (MP3 audio decoding)

### PR DESCRIPTION
**Pull request by dschreij**

### CODE REVIEW

**Comment by dschreij** [comment]
@conda-forge/core I can't provide tests for this library as it needs an audio environment which the CI's don't provide. Additionally, I have only compiled the static library because that is all I needed for building sdl2_mixer. Would you like me to build the dynamic library as well?

**Comment by scopatz** [comment]
LGTM, just need to test on linux.

**Comment by dschreij** [comment]
I noticed CircleCI disappeared, and heard its due to a 'bugfix' at their side. I haven't succeeded to reactivate it with tip I got to go to the main page and select to watch my recipes again. Is there anyway to reactivate the Linux CI's again?

File: recipes/mad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{% set name = "mad" %}
+{% set version = "0.15.1b" %}
+{% set sha256 = "bbfac3ed6bfbc2823d3775ebb931087371e142bb0e9bb1bee51a76a6e0078690" %}
+
+package:
+  name: lib{{ name|lower }}
```
Comment (jakirkham) [inline recipes/mad/meta.yaml]: The name of the package is `libmad` however the directory the recipe is in is `mad`. Could you please send a follow-up PR to staged-recipes to fix this discrepancy at @dschreij ? Currently this recipe (and a few others) are stuck in limbo until that happens.

**Pull request merged**